tika-core/src/main/java/org/apache/tika/config/TikaConfig.java

```diff
@@ -681,7 +681,7 @@ T loadOne(Element element, MimeTypes mimeTypes, ServiceLoader loader)
                     // Regular class, create as-is
                     loaded = newInstance(loadedClass);
                     // TODO Support arguments, needed for Translators etc
-                    // See the thread "Configuring parsers and translators" for details 
+                    // See the thread "Configuring parsers and staticTranslators" for details 
```

Comment:
Undo

Reply:
funny this sneaked in.. eclipse has its own life